### DISCUSSION TITLE
GRID_STAT_REGRID_TO_GRID set to a specific grib2 file

### DISCUSSION THREAD

**Discussion by BinbinZhou-NOAA** [dtcenter/METplus#2401] [Configuration] [answered]
In the RRFS ensemble verification over Alaska,  we try to set the verification on the same grid as the HREF's Alaska grid.
But HREF's Alaska grid  is not a standard.  So we have to use one of  the HREF's grib2 files over Alaska. In this case, 
GRID_STAT_REGRID_TO_GRID = href.alaska.grib2  in the METPLUS conf file, 
where  fref.alaska.grib2 is grib2 file name. 
Is this correct?
Thanks!

**Comment by JohnHalleyGotway** [comment, marked as answer]
@BinbinZhou-NOAA I see that you'd like to specify the desired verification grid by providing a GRIB2 file with data already on that grid.

Yes, you're correct, the METplus [GRID_STAT_REGRID_TO_GRID](https://metplus.readthedocs.io/en/latest/Users_Guide/glossary.html#term-GRID_STAT_REGRID_TO_GRID) configuration option provides that functionality. That METplus option sets MET's [regrid.to_grid](https://met.readthedocs.io/en/latest/Users_Guide/config_options.html#regrid) option which supports:
- to_grid = “path”; To regrid both to a grid defined by a file.

A couple of other thoughts.
- I'd recommend providing the full path to that `href.alaska.grib2` file by including `{OUTPUT_BASE}` or something similar in the path.
- Consider also setting the [GRID_STAT_REGRID_METHOD](https://metplus.readthedocs.io/en/latest/Users_Guide/glossary.html#term-GRID_STAT_REGRID_METHOD) and [GRID_STAT_REGRID_WIDTH](https://metplus.readthedocs.io/en/latest/Users_Guide/glossary.html#term-GRID_STAT_REGRID_WIDTH) options to specify the regridding method and width to be used. The default is nearest neighbor. To override that to bilinear, for example, set:
```
GRID_STAT_REGRID_METHOD = BILIN
GRID_STAT_REGRID_WIDTH = 2
```
You can find the supported options listed in the [regrid dictionary documentation](https://met.readthedocs.io/en/latest/Users_Guide/config_options.html#regrid).

**Reply by BinbinZhou-NOAA** [reply]
John,

   Thanks!

Binbin

**Comment by BinbinZhou-NOAA** [comment]
John,

  Thanks!

Binbin
